backend/app/controllers/users.rb

```diff
@@ -80,6 +81,22 @@ class ArchivesSpaceService < Sinatra::Base
   end
 
 
+  Endpoint.post('/users/recover-password')
+    .description("Initiate a password reset process by sending a one-time token to the user")
+    .params(["email", String, "The requestor's email address"])
+    .permissions([])
+    .returns([200, "password recovery email sent"]) \
+  do
+    raise NotAllowed.new unless AppConfig[:allow_password_reset]
+    user = User.find(email: params[:email])
```

Comment:
Should or could this be restricted to users with DBAuth? Or how would this behave in a SSO scenario where users do not have real passwords in the database like e.g. with https://github.com/lyrasis/aspace-oauth or https://github.com/nebulon42/aspace-oidc?

Reply:
This is a good point. I think for now it will probably be on admins to not turn on password reset (the default) if they use SSO/external authentication.